README.md

```diff
@@ -79,6 +79,21 @@ persistenceCore start
 * Start node `persistenceCore start`
 * To start rest server set `enable=true` in `config/app.toml` under `[api]` and restart the chain
 
+### Ledger Support( Experimental)
+
+> NOTE: This is an experimental feature. Persistence uses coin-type 750; generating keys through this method below will create keys with coin-type 118(cosmos) and will only be supported by CLI and not by current or future wallets. Ledger support for the Persistence application is coming soon.  
+
+* Install and open the Cosmos application on the Ledger device. [ref](https://hub.cosmos.network/main/resources/ledger.html#install-the-cosmos-ledger-application)
+* Connect the Ledger device to the terminal.
```

Comment:
just need to connect device to the system, have cosmos app `on` in the ledger, only then the commands work.